android/src/main/java/org/devio/rn/splashscreen/SplashScreen.kt

```
@@ -0,0 +1,108 @@
+                val lottie = mSplashDialog?.findViewById<LottieAnimationView>(lottieId)
+
+                lottie?.addAnimatorListener(object : Animator.AnimatorListener {
+                    override fun onAnimationStart(animation: Animator) {
+                        println("SplashScreen is started")
+                    }
+
+                    override fun onAnimationEnd(animation: Animator) {
+                        setAnimationFinished(true)
+                    }
+
+                    override fun onAnimationCancel(animation: Animator) {}
```

detekt

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)